tests/sanity/ignore-2.12.txt

```diff
@@ -0,0 +1,13 @@
+automation/build.sh shebang!skip
+build.sh shebang!skip
+changelogs/fragments/.gitignore changelog!skip
```

Comment:
Why do we need .gitignore under changelogs/fragments? Wouldn't it be better to remove it?
https://github.com/oVirt/ovirt-ansible-collection/pull/398